cmd/addvalidators.go

```diff
@@ -137,12 +151,22 @@ func runAddValidatorsSolo(_ context.Context, conf addValidatorsConfig) (err erro
 	}
 
 	// Finally, perform a cluster transformation.
-	_, err = manifest.Transform(cluster, addVals)
+	cluster, err = manifest.Transform(cluster, addVals)
 	if err != nil {
 		return errors.Wrap(err, "transform cluster manifest")
 	}
 
-	// TODO(xenowits): Write new cluster manifest to disk, see issue https://github.com/ObolNetwork/charon/issues/1887.
+	// Save cluster backup to disk
+	err = writeClusterBackup(conf.ClusterBackupDir, oldCluster)
```

Comment:
this doesn't make sense if you loaded from legacy lock file, it only makes sense if loaded from manifest file.

Reply:
Good point. But, removing the cluster-backup stuff from this PR, will add this in the subsequent PR for ticket: https://github.com/ObolNetwork/charon/issues/2345